otter/convergence/gathering.py

```diff
@@ -211,9 +210,9 @@ def update_drained_at(node):
         if node.description.lb_id in deleted_lbs:
             return None
         if feed is not None:
-            return assoc_obj(node, drained_at=extract_clb_drained_at(feed))
-        else:
-            return node
+            node.drained_at = extract_clb_drained_at(feed)
```

Comment:
> Why is this being changed to mutate its input rather than return a new value?

I didn't want to do call `assoc` with a private variable. 

> What is the input being shared with?

What input? `node` argument? Sorry, didn't understand.